src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class ParserTest {
+
+    @Test
+    public void parse_bye_success() throws DukeException{
+        assertEquals("", new Parser(null).parse("88"));
+    }
+    @Test
+    public void parse_undefinedInput_exceptionThrown() {
```

Comment:
Good adherence to the "featureUnderTest_testScenario_expectedBehavior()" as per the coding standard for test cases.

Reply:
yep i agree with this review comment!